dev/run-python-flavor-tests.sh

```diff
@@ -17,6 +17,7 @@ pytest --verbose tests/azureml --large
 pytest --verbose tests/models --large
 pytest --verbose tests/xgboost --large
 pytest --verbose tests/lightgbm --large
+pytest --verbose tests/catboost --large
```

Comment:
Do we need add cross version test for catboost ?

Reply:
We do. I was planning to add that in a follow-up PR.